test/ably/rest/restchannelpublish_test.py

```
@@ -402,26 +401,31 @@ async def test_interoperability(self):
-                # temporary added delay, we need to investigate why messages don't appear immediately
-                await asyncio.sleep(1)
-                async with httpx.AsyncClient(http2=True) as client:
-                    r = await client.get(url, auth=auth)
-                item = r.json()[0]
-                assert item.get('encoding') == encoding
-                if encoding == 'json':
-                    assert json.loads(item['data']) == json.loads(msg_data)
-                else:
-                    assert item['data'] == msg_data
+                response = await channel.publish(data=expected_value)
+                assert response.status_code == 201
```

Ruff (0.12.2)

Use of `assert` detected

(S101)